tests/get-some-time/pass_afonso.js

```diff
@@ -0,0 +1,32 @@
+Date.prototype.addDays = function (days) {
```

Comment:
don't add methods to prototype it's not a good practice because they leak everywhere (it's not scoped to your file)

```js
const addDays = (d, days) => {
  const date = new Date(d.valueOf())
  date.setDate(d.getDate() + days)
  return date
}
```
Don't just copy / past shit from stackoverflow when doing javascript or you will end up in pain haha